pkg/metadata/host/host_windows.go

```diff
@@ -5,14 +5,163 @@
 
 package host
 
-import "github.com/shirou/gopsutil/host"
+import (
+	"fmt"
+	"os"
+	"runtime"
+	"strconv"
+	"strings"
+	"syscall"
+	"time"
 
+	"github.com/DataDog/datadog-agent/pkg/metadata/common"
+	"github.com/DataDog/datadog-agent/pkg/util/cache"
+	"github.com/DataDog/datadog-agent/pkg/util/winutil"
+	"github.com/DataDog/gohai/cpu"
+	"github.com/DataDog/gohai/platform"
+	"github.com/shirou/w32"
+)
+
+var (
+	modkernel = syscall.NewLazyDLL("kernel32.dll")
+
+	procGetTickCount64 = modkernel.NewProc("GetTickCount64")
+)
+
+// InfoStat describes the host status.  This is not in the psutil but it useful.
+type InfoStat struct {
+	Hostname             string `json:"hostname"`
+	Uptime               uint64 `json:"uptime"`
+	BootTime             uint64 `json:"bootTime"`
+	Procs                uint64 `json:"procs"`           // number of processes
+	OS                   string `json:"os"`              // ex: freebsd, linux
+	Platform             string `json:"platform"`        // ex: ubuntu, linuxmint
+	PlatformFamily       string `json:"platformFamily"`  // ex: debian, rhel
+	PlatformVersion      string `json:"platformVersion"` // version of the complete OS
+	KernelVersion        string `json:"kernelVersion"`   // version of the OS kernel (if available)
+	VirtualizationSystem string `json:"virtualizationSystem"`
+	VirtualizationRole   string `json:"virtualizationRole"` // guest or host
+	HostID               string `json:"hostid"`             // ex: uuid
+}
 type osVersion [2]string
 
 //Set the OS to "win32" instead of the runtime.GOOS of "windows" for the in app icon
 const osName = "win32"
 
-func fillOsVersion(stats *systemStats, info *host.InfoStat) {
+// Collect at init time
+var cpuInfo []InfoStat
```

Comment:
same

Reply:
this is how the metadata is populated; not sure what you mean by 'same'

Reply:
(not a blocker, see other comment)